resources/views/layouts/master.blade.php

```diff
@@ -50,6 +50,9 @@
                             <li class="nav-item">
                                 <a class="nav-link" href="{{ route('login') }}">{{ __('Login') }}</a>
                             </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="{{ url('/info') }}">{{ __('Info') }}</a>
```

Comment:
I would put this link on the left side of the nav bar... It seems a little weird to have it between the login and register buttons. Also, instead of `{{ url('/info') }}` use `{{ route('info') }}` (or whatever you name the route in the web.php - see my other comment). I know it might seem trivial, but if we want to change the route url later (let's say to /water-info, then we only need to update it in the web.php route file and not here as well.